docs/source/chaincode4ade.rst

```diff
@@ -39,11 +39,17 @@ In each language, the ``Invoke`` method is called by clients to submit transacti
 proposals. This method allows you to use the chaincode to read and write data on
 the channel ledger.
 
-You also need to include an ``Init`` method that will serve as the initialization
-function for your chaincode. This method will be called in order to initialize
-the chaincode when it is started or upgraded. By default, this function is never
-executed. However, you can use the chaincode definition to request that the ``Init``
-function be executed. If execution of ``Init`` is requested, fabric will ensure
+If you use the low level APIs provided by the Fabric Chaincode Shim API,
+you need to include an ``Init`` method in your chaincode that will serve as the
+initialization function. You can use this method to implement logic that is required
+to initialize your chaincode when it is started or upgraded. By default, this function is never
+executed. However, you can use the chaincode definition to request the execution
```

Comment:
"request the execution" makes it sound like Init will be called automatically.
Mention that the definition indicates whether Init must be called by a user, after the definition has been committed, but before regular Invokes.

Also, clarify that init-required is applicable only when the 'version' increments in the definition.